Documentation/specs/runtime-configuration-file.md

```diff
@@ -207,7 +207,7 @@ This section is only present in the root framework's (so `Microsoft.NETCore.App`
 
 The file is read by two different components:
 
-* The host uses it to determine what to place on the TPA and Native Library Search Path lists, as well as what runtime settings to apply (GC type, etc.). See [the host spec](corehost.md).
+* The host uses it to determine what to place on the TPA and Native Library Search Path lists, as well as what runtime settings to apply (GC type, etc.). See the host [resolution](https://github.com/dotnet/core-setup/blob/master/Documentation/design-docs/host-component-dependencies-resolution.md) and [probing](https://github.com/dotnet/core-setup/blob/master/Documentation/design-docs/host-probing.md) documentation.
```

Comment:
The `host-component-dependencies-resolution.md` document is not describing the general handling of `.deps.json`. I'm afraid we don't really have a detailed spec for that - this document you're modifying is all we've got. The document you refer to here described the specifics of dependency resolution which is currently used by the `AssemblyDependencyResolver` (it used to be called `ComponentDependencyResolver`.

Reply:
I agree, I tried to add the closest replacement for the non-existant reference to `corehost.md` file.
The dependency resolution/probing docs were the closest ones which process deps.json files and load assemblies. 

I can remove the reference to these files if you think that's better.

Reply:
I changed it in this checkin: ca07a5d